api/util/registerUser.js

```diff
@@ -0,0 +1,130 @@
+const User = require('../models/User');
+const config = require('../config/config');
+
+/**
+ * Register a new user.
+ * @param {Object} newUser - The user that is to be registered
+ * @param {(string|undefined)} - The information of the User
+ *                               that is to be added
+ */
+
+
+async function registerUser(req){
```

Comment:
can you make the variable name more descriptive
```suggestion
async function registerUser(userToAdd){
```